maps/biter_battles_v2/main.lua

```diff
@@ -11,6 +11,9 @@ local Team_manager = require "maps.biter_battles_v2.team_manager"
 local Terrain = require "maps.biter_battles_v2.terrain"
 local Session = require 'utils.datastore.session_data'
 local Color = require 'utils.color_presets'
+local autoTagWestOutpost="[WestOutpost]"
+local autoTagEastOutpost="[EastOutpost]"
+local autoTagDistance=600
```

Comment:
```suggestion
local autoTagWestOutpost = "[WestOutpost]"
local autoTagEastOutpost = "[EastOutpost]"
local autoTagDistance = 600
```
To be consistent with other parts of the code

Reply:
updated it